src/apps/answers/domain/answers.py

```diff
@@ -238,7 +237,6 @@ class AssessmentAnswerPublic(PublicModel):
     answer: str | None
     item_ids: list[str] = Field(default_factory=list)
     items: list[PublicActivityItemFull] = Field(default_factory=list)
```

Comment:
This field is populated somewhere. Please remove this logic as well. It's calculated like date_created != date_updated